runtime/caml/addrmap.h

```diff
@@ -0,0 +1,99 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*                Stephen Dolan, University of Cambridge                  */
+/*                                                                        */
+/*   Copyright 2020 Indian Institute of Technology, Madras                */
+/*   Copyright 2020 University of Cambridge                               */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+#include "mlvalues.h"
+
+#ifndef CAML_ADDRMAP_H
+#define CAML_ADDRMAP_H
+
```

Comment:
The whole of this can probably be protected with `#ifdef CAML_INTERNALS`, I think?

Reply:
Updated in 4e084b5721eabeea8d75214bf811d9e0a651b1e5.